### DISCUSSION TITLE
how to set screen scale from dbus or cmdline

### DISCUSSION THREAD

**Discussion by ihipop** [linuxdeepin/developer-center#2988] [Q&A | 问答板块]
any command or dbus address like dconf to set screen scale？

**Comment by ihipop** [comment]
as far as i know i can use these commands

https://github.com/linuxdeepin/startdde/blob/10bdd0930cab4db6179a7fbc18aaa2654d46a853/xsettings/scale.go#L50-L52


```
gsettings set com.deepin.xsettings scale-factor 2
gsettings set com.deepin.xsettings window-scale 2
gsettings set com.deepin.xsettings gtk-cursor-theme-size 48
```

but it does not send any dbus messages

**Comment by Zeno-sole** [comment]
Maybe xrandr can help you
